### ISSUE TITLE
Issue with react peerDependencies?

### ISSUE THREAD

**Issue by sebastienbarre** [saulhoward/react-evil-icons#1]
Hi
I was wondering if the ```peerDependencies``` could be somewhat updated, maybe to ```>=0.12.2```?
I'm using react 0.13, and trying to install ```react-evil-icons```:
```
npm WARN peerDependencies The peer dependency react@^0.12.2 included from react-evil-icons will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm ERR! Darwin 14.1.0
npm ERR! argv "node" "/Volumes/Users/barre/.npm-packages/bin/npm" "install" "--save" "react-evil-icons"
npm ERR! node v0.12.0
npm ERR! npm  v2.7.3
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package react does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer fixed-data-table@0.1.2 wants react@>=0.12.0 <0.14.0
npm ERR! peerinvalid Peer react-router@0.13.2 wants react@0.13.x
npm ERR! peerinvalid Peer react-hot-loader@1.2.4 wants react@>=0.11.0
npm ERR! peerinvalid Peer react-evil-icons@0.3.0 wants react@^0.12.2
```
I think there is a clash with other packages' ```peerDependencies```, though I might be wrong.
Maybe at the same rate ```6to5``` could be updated to ```babel```, since it has replaced ```6to5```?
Note that ```jest``` is pretty unstable at the moment, so it's possible the ```prepublish``` step is going to fail because of ```npm test```.
Thank you.

**Comment by saulhoward** [comment]
Great points, thank you! I've updated to babel and used the react peerDependency `>=0.12`.

This is now published as version 0.4.0 on npm.

**Comment by sebastienbarre** [comment]
Thanks!
